main.js

```diff
@@ -0,0 +1,35 @@
+function generateNumbers() {
+    let numbersOnScreen = Math.floor(Math.random() * 3) + 3;
+    let allNumbers = [];
+
+    for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = Math.floor(Math.random() * 101);
+
+        allNumbers.push(randNumber);
+    }
+
+    return allNumbers;
+}
+
+function displayNumbers(allNumbers) {
+    let numbersOnScreen = allNumbers.length;
+
+     // using a for loop instead of forEach as the array.pop changes the length over each iteration
+     for (let i = 0; i < numbersOnScreen; i++) {
+        let randNumber = allNumbers.pop();
+        let number = document.createElement('div');
+
+        number.textContent = randNumber;
+        number.setAttribute('class', 'numberBox');
+        document.querySelector('#play_area').appendChild(number);
+    }
+
+}
+
+document.querySelector('#start').addEventListener('click', () => {
+    document.querySelector('#splash').style.display = 'none';
+    document.querySelector('#game').style.display = 'block';
+    timer();
+    let randomNumbers = generateNumbers();
+    displayNumbers(randomNumbers);
```

Comment:
Personally, I would organise the arrow function slightly better, as it groups related code together. Also, probably better off switching the `let` to a `const` as we don't want to be able to change the contents.
```suggestion
    const randomNumbers = generateNumbers();
    
    document.querySelector('#splash').style.display = 'none';
    document.querySelector('#game').style.display = 'block';
   
    displayNumbers(randomNumbers);
    timer();
```

Reply:
we literally just added `Timer();` but we will change the other story stuff.

layout is a preference thing but thanks.

Reply:
```suggestion
    const randomNumbers = generateNumbers();
    displayNumbers(randomNumbers);
    document.querySelector('#splash').style.display = 'none';
    document.querySelector('#game').style.display = 'block';
    timer();
```
I reckon this would be the way that makes most sense. Have the numbers generated and then added to html ready to be display -> display the game page -> start the timer.